util/log.go

```diff
@@ -2,10 +2,13 @@ package util
 
 import (
 	"io"
+	"log/syslog"
 	"os"
 	"path/filepath"
+	"slices"
 
 	log "github.com/sirupsen/logrus"
+	lSyslog "github.com/sirupsen/logrus/hooks/syslog"
```

Comment:
The syslog package is not available for Windows. We need to split the code that uses it into:

syslog_nonwindows.go <- with build tags like: `// +build !windows` and with a function that initializes the log with the syslog hook
syslog_windows.go <- with a function that does nothing